### PR TITLE
ref: Remove reference to deprecated Event model

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

**Comment by lynnagara** [comment]
@getsentry/ecosystem I'm not too sure which plugins this might affect? Which plugins are the v1 / v2 ones?

File: src/sentry/plugins/base/v1.py [hunk 1]
```diff
@@ -320,8 +318,9 @@ def get_view_response(self, request, group):
         if not isinstance(response, Response):
             raise NotImplementedError("Use self.render() when returning responses.")
 
-        event = group.get_latest_event() or Event()
-        event.group = group
+        event = group.get_latest_event()
```
Comment (fpacifici) [inline src/sentry/plugins/base/v1.py]: The event instance is returned in the response. So now it can be None while in the old implementation there was always an object returned. Is this change intentional ?

**Review by fpacifici — CHANGES_REQUESTED** [review]
Please verify whether returning None is fine.

**Comment by lynnagara** [comment]
From what I can tell, the only place that uses this (has custom view function) is the teamwork plugin and when I try to get to that page I get InternalServerError anyway https://sentry.io/sentry/sentry/issues/1162336372/actions/teamwork/. So I don't think this breaks anything further than it is already broken :/

**Review by fpacifici — APPROVED** [review]
